src/components/Header.js

```diff
@@ -0,0 +1,139 @@
+import React from "react";
```

Comment:
extracting the header into a separate file should really have been a separate pull request